src/quoi-feur.ts

```diff
@@ -0,0 +1,100 @@
+import { type ChatInputCommandInteraction, type Message, TextChannel } from 'discord.js';
+
+import { MUTED_BY_BOT } from './constants/roles';
+import { cache } from './helpers/cache';
+
+const quoiDetector = new RegExp(/\b\s*[qQ][uU][oO][iI]\s*[.,!?]*\s*$/i);
```

Comment:
can we replace that using a function that checks using `String#lowercase` ?
so that we don't use a regex

Reply:
Ton `/i` a la fin de la RegExp n'est pas déjà ce qui te permet de faire une vérification insensible à la casse ? Si oui, pourquoi avoir `[qQ][uU][oO][iI]` ?

Reply:
good point